docfx_project/articles/bootstrapping-trust/part-04.certificate-authorities-and-chains-of-trust.md

```diff
@@ -57,45 +60,48 @@ chicken-and-egg problem. Trust for CAs has to start somewhere. With CAs
-should be rare, there is less information to distribute about who not to
-trust rather than who to trust from a particular Root CA.
+Compromised intermediate CAs can be black listed.
 
 Running a public CA is serious business if you wish to be publicly
-trusted. The organization has to have strict protocols that verify the
-security and safe handling of the CAs private keys. If the private key
-is compromised, it can be used to sign other certificates for malicious
-intents. Any system that trusted the CA will now trust any of the
-maliciously signed certificates. This can, and most likely will,
-compromise all encryption on all connections that trusted the CA.
+trusted. The organization's running a CA have to have strict protocols
+that verify the security and safe handling of the CAs private keys. If
+the private key is compromised, it can be used to sign other
+certificates for malicious intents. Any system that trusted the
+compromised CA will now trust any maliciously signed certificates. This
+will compromise all certificates singed by that CA.
 
 Public CAs are maintained by organizations such as DigiCert, Let's
-Encrypt, and others. Anyone can also create private CAs. The only
-difference is that the number of systems that trust a private CA is much
-smaller than that of a public one. CAs are a cornerstone of
-bootstrapping trust in a software system public or private.
+Encrypt, and others. Anyone can create private CAs. The only difference
+is that the number of systems that trust a private CA is much smaller
+than that of a public one. CAs are a cornerstone of bootstrapping trust.
+Trusting the proper CAs can grant trust to a large number of systems.
 
 ### Chains of Trust & PKIs
 
-Part three of this series introduced that certificates could be signed
-by itself or another certificate. Certificates are usually signed via
-Certificate Signing Requests (CSRs). A certificate signing itself is
-called a "self-signed certificate" and is an indicator of it being a
-root CA if the CA flag is also set to true. A root CA can sign other
-certificates with the CA flag set to true, and those become intermediate
-CAs. Any CA, root or intermediate, that fulfills a CSR and signs the
-enclosed certificate will generate a non-CA certificate as long as the
-CA flag is false. These certificates are leaf certificates.
-
-Public Key Infrastructure (PKI) is used to describe the output of
-configuring and generating all of the CAs and processes needed to
-create, manage, and verify trust for leaf certificates.
+[Part three](./part-03.certificates.md) of this series introduced that
+certificates self-sign or sign another certificate. Certificates are
+usually signed via Certificate Signing Requests (CSRs). A certificate
+signing itself is called a "self-signed certificate" and is an indicator
+of it being a root CA if the CA flag is also set to true. A root CA can
+sign other certificates that also have the CA flag set to true. Those
+types of  
```

Comment:
if possible - consider removing this line feed and formatting the remaining lines. if not - meh